jung-algorithms/src/main/java/edu/uci/ics/jung/layout/algorithms/FRBHLayoutAlgorithm.java

```diff
@@ -119,7 +127,6 @@ private void doInit() {
       repulsion_constant = repulsion_multiplier * forceConstant;
       initialized = true;
       tree = new BarnesHutQuadTree(layoutModel);
-      tree.rebuild();
     }
```

Comment:
no reason to rebuild until the layout algorithm steps.

Reply:
I'm not sure that's true.  The BHQT constructor doesn't actually build the tree, it just does some initial setup.  it's possible that we don't need it yet at this point, but we're going to have to pay that cost at some point.